Core/include/Acts/Seeding/detail/CylindricalSpacePointGrid.hpp

```diff
@@ -56,9 +57,9 @@ struct CylindricalSpacePointGridConfig {
   // maximum impact parameter in mm
   float impactMax = 0 * Acts::UnitConstants::mm;
   // minimum phi value for phiAxis construction
-  float phiMin = -M_PI;
+  double phiMin = -std::numbers::pi;
```

Comment:
@andiwand @paulgessinger concerning this. What is your opinion? `double` or `float`?
We can have as well `std::numbers::pi_v<float>` I believe.

I'd eventually go to `double` since `mm` and units are `double` ... but moving everything to `double` will be quite an extensive change in the end.

Reply:
If we really do all the calculations here with `float` I would stick with that type to avoid runtime conversion

Reply:
done